packages/app/src/app/pages/common/Modals/PreferencesModal/index.tsx

```diff
@@ -25,12 +26,18 @@ import { Experiments } from './Experiments';
 import { Integrations } from './Integrations';
 import { KeyMapping } from './KeyMapping';
 import { PaymentInfo } from './PaymentInfo';
+import { MailPreferences } from './MailPreferences';
+
 import { SideNavigation } from './SideNavigation';
 
 type MenuItem = ComponentProps<typeof SideNavigation>['menuItems'][0] & {
   Content: ComponentType;
 };
-const getItems = (isLoggedIn: boolean, isPatron: boolean): MenuItem[] =>
+const getItems = (
+  isLoggedIn: boolean,
+  isPatron: boolean,
+  user: any
```

Comment:
You can make this `CurrentUser` (available in `@codesandbox/common/lib/types`).

Reply:
Done :)